src/aquarium.py

```diff
@@ -69,28 +69,28 @@ async def aquarium_startup(aquarium_app: FastAPI, aquarium_api: FastAPI):
 
     # Set up all of the tickers
     devices: Devices = Devices(
-        gstate.config.options.devices.probe_interval,
-        nodemgr,
-        ceph_mgr,
-        ceph_mon
+        gstate.config.options.devices.probe_interval, nodemgr, ceph_mgr, ceph_mon
```

Comment:
gotta say, this annoys me a lot. I think it makes readability worse.

Reply:
in retrospect, I think I can live with this sort of line wrapping. It's not what I'd prefer, but in the end `black` seems to be doing the right thing if even that line is too big:

```
# in:

def very_important_function(template: str, *variables, file: os.PathLike, engine: str, header: bool = True, debug: bool = False):
    """Applies `variables` to the `template` and writes to `file`."""
    with open(file, 'w') as f:
        ...

# out:

def very_important_function(
    template: str,
    *variables,
    file: os.PathLike,
    engine: str,
    header: bool = True,
    debug: bool = False,
):
    """Applies `variables` to the `template` and writes to `file`."""
    with open(file, "w") as f:
        ...

```

Reply:
I don't disagree. There are some styles of black that don't read as well to me, but I'm happy with that trade off for otherwise strict and reasonably sensible enforcement.